casatables/src/glue.h

```diff
@@ -89,9 +90,23 @@ typedef enum TableOpenMode {
     TOM_CREATE = 3,
 } TableOpenMode;
 
+typedef enum TableCreateMode {
+    // create table
+    TCM_NEW = 1,
+	// create table (may not exist)
```

Comment:
Indentation here got a little wonky somehow.

Reply:
Oops, I copied and pasted something from casacore. Fixed!